src/data/matrix/basic.lean

```diff
@@ -574,6 +574,18 @@ lemma sum_apply [add_comm_monoid α] (i : m) (j : n)
   (∑ c in s, g c) i j = ∑ c in s, g c i j :=
 (congr_fun (s.sum_apply i g) j).trans (s.sum_apply j _)
 
+lemma mat_two_mul_expl {R : Type*} [comm_ring R] (A B : matrix (fin 2) (fin 2) R) :
```

Comment:
I guess we are already in some `matrix` namespace, so `mat` in the name is useless.

Reply:
Sure I'll remove the mat.